test/config/jest.image.ts

```diff
@@ -59,78 +59,99 @@ class RetriesCounter {
 
 const retriesCounter = new RetriesCounter();
 
-async function saveAndRegisterImages(matcherContext: MatcherContext, options: MatchImageSnapshotOptions): Promise<void> {
-  const snapshotIdentifier = options.customSnapshotIdentifier as string;
-  // Manage expected and received images
-  const baseImagePathWithName = `${options.customDiffDir}/${snapshotIdentifier}`;
-  const expectedImagePath = `${baseImagePathWithName}-expected.png`;
-  copyFileSync(`${options.customSnapshotsDir}/${snapshotIdentifier}.png`, expectedImagePath);
-  // this image is generated by jest-image-snapshot when activating `storeReceivedOnFailure`
-  const receivedImagePath = `${baseImagePathWithName}-received.png`;
-
+async function attachImagesForReport(images: LocationOfImagesForTestReport, matcherContext: MatcherContext): Promise<void> {
   // Attach the images to jest-html-reports
   // Chain the calls to preserve the attachment order
   // Create a custom context as the async call can be done whereas the global jest context has changed (another test is currently running).
   // So the test name and path changed, and the images would be attached to the wrong test.
   // For the context object structure, see https://github.com/Hazyzh/jest-html-reporters/blob/v3.0.5/helper.ts#L95
   const context: Record<symbol, unknown> = {};
+  const currentTestName = matcherContext.currentTestName;
+  log('Attaching images to report for test %s', currentTestName);
   context[Symbol('bpmn-visualization')] = {
     state: {
-      currentTestName: matcherContext.currentTestName,
+      currentTestName: currentTestName,
```

Comment:
**suggestion:** 
```suggestion
      currentTestName,
```

Reply:
Good catch, thanks.
I will integrate this prior merging.

Reply:
Fixed in f579926